v3/src/components/graph/utilities/pixi-points.ts

```diff
@@ -283,15 +290,30 @@ export class PixiPoints {
   }
 
   getPointTexture(style: IPixiPointStyle): PIXI.Texture {
-    const { radius, fill, stroke, strokeWidth, strokeOpacity } = style
-    const key = this.textureKey(style)
+    // TODO: It would be better to not create new textures for every width and height as doing so will generate more 
+    // textures than necessary, possibly causing performance issues. If possible, generate a single texture here, and
+    // when it's necessary to vary width and height (i.e. when displayType === "bars"), adjust those properties
+    // directly on the sprites elsewhere, probably in `setPointStyle`. There are scaling issues to do with stroke width
+    // that need to be worked out before we can do that, though. For now, we generate a new texture for each width
+    // and height when `displayType` is `bars`.
+    const { radius, fill, stroke, strokeWidth, strokeOpacity, width, height } = style
+    const styleForKey = this.displayType !== "bars"
+      ? { radius, fill, stroke, strokeWidth, strokeOpacity }
+      : style
+    const styleAndDisplayType = { ...styleForKey, displayType: this.displayType }
+    const key = this.textureKey(styleAndDisplayType)
+    // If there's already a matching texture, return that instead of creating a new one.
     if (this.textures.has(key)) {
       return this.textures.get(key) as PIXI.Texture
     }
     const graphics = new PIXI.Graphics()
     graphics.beginFill(fill)
     graphics.lineStyle(strokeWidth, stroke, strokeOpacity ?? 0.4)
-    graphics.drawCircle(0, 0, radius)
+    if (this.displayType === "bars" && width && height) {
+      graphics.drawRect(0, 0, width, height)
+    } else {
+      graphics.drawCircle(0, 0, radius)
+    }
```

Comment:
Minor point because we presumably don't expect it to happen, but if there is no width or height it seems like it might be better to draw a rectangle by default, so perhaps this should be something like:
```suggestion
    if (this.displayType === "bars") {
      graphics.drawRect(0, 0, width ?? radius, height ?? radius)
    } else {
      graphics.drawCircle(0, 0, radius)
    }
```